Gemfile.lock

```diff
@@ -0,0 +1,43 @@
+PATH
```

Comment:
Woah, upstream doesn't have a Gemfile.lock commited?  That's super strange.  I think you're doing the right thing here by locking it down.

Reply:
Cool. I'd wondered as well.